README.md

```diff
@@ -1,2 +1,109 @@
-# bamboo-viability
-🎍 Findings from our EVE Bamboo I viability assessment
+# Bamboo I Viability Assessment
```

Comment:
Feel like you might want to deliver this as a Google Doc on our NA letter head.

Reply:
@lleger Will do once I've got all of the comments here resolved.